tests/test_convert.py

```diff
@@ -1224,3 +1231,34 @@ def test_hardware_mode_and_user_gates():
     expected = QuantumCircuit([Qubit()])
     expected.u(0, 4.5, 0, 0)
     assert qc.data[1].operation.definition == expected
+
+
+def test_box_statements():
+    # pylint: disable=no-member
+
+    source = """
+        include 'stdgates.inc';
+        qubit a;
+        qubit b;
+
+        box {
+            x a;
+        }
+        box[1.0ms] {
+            box[2dt] {
+                cz a, b;
+            }
+        }
+    """
+    if QISKIT_VERSION < (2, 0):
```

Comment:
Are you running with  old qiskit in CI anywhere or is this just for local testing?

Reply:
The CI on this repo is still set up to test against `pip install qiskit` and (effectively) `pip install github.com/Qiskit/qiskit@main`.  So at the time the CI jobs ran, it tested against both, but now I'll need to fix the CI matrix to specifically test `'qiskit<2'` and `'qiskit>=2'`.